modules/ixBidAdapter.js

```diff
@@ -475,7 +473,23 @@ export const spec = {
       type: 'iframe',
       url: USER_SYNC_URL
     }] : [];
+  },
+
+  /**
+   * @param {BidRequest} bidRequest
+   * @param bidderRequest
+   * @returns {string}
+   */
+  _getPageUrl: function (bidRequest, bidderRequest) {
+    let pageUrl = config.getConfig('pageUrl');
+    if (bidRequest.params.referrer) {
+      pageUrl = bidRequest.params.referrer;
+    } else if (!pageUrl) {
+      pageUrl = bidderRequest.refererInfo.referer;
+    }
```

Comment:
Since we want to use always our canonical, I think it's ok to rely on `config.getConfig('pageUrl')`.
```suggestion
```

Reply:
Fixed